media/css/firefox/whatsnew/whatsnew-modular.scss

```diff
@@ -0,0 +1,288 @@
+        color: $color-off-black;
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Extra content
+
+.c-block {
+    @include border-box;
+    margin: 0 auto $spacing-2xl;
+    max-width: $content-md - ($layout-md * 2);
+    padding: 0 $layout-md;
+
+    .c-block-title {
+        @include text-display-xs;
+    }
+
+    .c-block-icon {
+        margin-bottom: $spacing-md;
+    }
+}
+
+@media #{$mq-md} {
+    .l-blocks-two {
+        display: flex;
+        margin: 0 auto;
+        max-width: $content-md - ($layout-md * 2);
+        padding: 0 $layout-md;
+
+        .c-block {
+            flex: 1 1 0;
+            padding: 0 $spacing-lg;
+        }
+    }
+}
+
+
+@media #{$mq-lg} {
+    .l-blocks-three {
+        display: flex;
+        margin: 0 auto;
+        max-width: $content-lg;
+
+        .c-block {
+            flex: 1 1 0;
+            padding: 0 $spacing-lg;
+        }
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// FxA Form
+
+.c-signup-container {
+    display: flex;
+    flex-direction: row;
+}
+
+.fxa-form-cta {
+    .fxa-email-form {
+        width: 100%;
+        margin-bottom: $spacing-lg;
+
+        a {
+            color: $color-off-black;
+        }
+    }
+
+    .fxa-email-form-intro {
+        display: none;
+    }
+
+    .fxa-email-field-container {
+        .field {
+            label {
+                @include visually-hidden;
+            }
+
+            input[type="email"] {
+                box-shadow: none;
+                padding: 12px 16px;
+                width: 100%;
+            }
+        }
+
+        .mzp-c-button {
+            width: 100%;
+        }
+    }
+
+    @media #{$mq-sm} {
+        .fxa-email-field-container {
+            @include grid-column-gap($spacing-md);
+            display: grid;
+            grid-template-columns: 2fr 1fr;
+
+            .field {
+                margin-bottom: 0;
+            }
+        }
+    }
+
+    @media #{$mq-md} {
+        .fxa-email-form {
+            display: grid;
+
+            .agreement {
+                @include text-body-sm;
+                grid-row-start: 2;
+                margin-top: $spacing-sm;
+                margin-bottom: 0;
+            }
+        }
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Dark theme
+
+.t-dark {
+
+    &.content-wrapper {
+        background-color: $color-off-black;
+        color: $color-white;
+    }
+
+    :link {
+        color: $color-white;
+
+        &:hover,
+        &:focus {
+            color: $color-pink-5;
+        }
+
+        &:active,
+        &:focus {
+            background-color: transparentize($color-white, .95);
+        }
+    }
+
+    :visited {
+        color: $color-pink-10;
+
+        &:hover,
+        &:focus {
+            color: $color-white;
+        }
+    }
+
+    .c-page-header {
+        background-color: $color-violet-90;
```

Comment:
nit: these two purples are _really_ close in color. I think they should have some visual separation.

![Screen Shot 2019-07-30 at 12 17 51 PM](https://user-images.githubusercontent.com/30483988/62158364-165c6980-b2c4-11e9-94c5-fd0207df7063.png)

I noodled on some quick CSS that I think follows our brand guidelines [gradient border]:
![Screen Shot 2019-07-30 at 1 07 29 PM](https://user-images.githubusercontent.com/30483988/62161461-1449d900-b2cb-11e9-8aa2-f5cd35bb0b07.png)


**HTML:**
```
  <header class="c-page-header">
    <div class="mzp-l-header-container">
      <div class="mzp-l-content c-page-header-inner">
      {% block header_logo %}
        {{ high_res_img('logos/firefox/logo-quantum-wordmark-large.png', {'alt': 'Firefox', 'width': '122', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
      {% endblock %}
        <h1 class="c-page-header-up-to-date"><span>{{ _('Congrats! You’re using the latest version of Firefox.') }}</span></h1>
      </div>
    </div>
  </header>
```

**CSS**
```
.t-dark {
    .c-page-header {
        background: linear-gradient(to right, $color-off-black, $color-violet-50, $color-red-50, $color-yellow-30);
        padding-bottom: 4px;

        .mzp-l-header-container {
            background-color: $color-off-black;
        }
    }
}
```

Reply:
I don't know about the rainbow border, and I don't want to go too far from the design that was already approved. But I do think we can make the demarcation clearer with a thin border in a slightly lighter purple, almost like a highlight on the edge.

Reply:
true true, I just had the styles kicking around. I think a thin slightly lighter border would be perfect.